profile-widget/profilewidget2.cpp

```diff
@@ -1610,7 +1610,7 @@ void ProfileWidget2::addDivemodeSwith()
 	QAction *action = qobject_cast<QAction *>(sender());
 	QPointF scenePos = mapToScene(mapFromGlobal(action->data().toPoint()));
 	for (i = 0; i < UNDEF_COMP_TYPE; i++)
-		if (QString(divemode_text[i]) == action->text())
+		if (QString(divemode_text_ui[i]) == action->text())
```

Comment:
Careful here - isn't this now comparing a untranslated string with an translated string?